shared/profile/user/index.js

```diff
@@ -202,6 +202,7 @@ export class BioTeamProofs extends React.PureComponent<BioTeamProofsProps> {
           onClick={this.props.onAddIdentity}
           style={styles.addIdentityButton}
           type="Secondary"
+          childrenRight={true}
         >
           <Kb.Meta backgroundColor={Styles.globalColors.blue} title="NEW" style={styles.newMeta} />
```

Comment:
Include the label as text in `children` and remove the `label` prop. Extra padding on the right is from `styles.newMeta`